lib/validations/etl.js

```diff
@@ -95,8 +104,20 @@ function subscribe(rippled) {
 
     if (data.type === 'validationReceived') {
       data.reporter_public_key = connections[this.url].public_key;
+
+      // Store master key if validation is signed
+      // by a known valid ephemeral key
+      const master_public_key = manifests.getMasterKey(
+        data.validation_public_key);
+      if (master_public_key) {
+        data.validation_public_key = master_public_key
```

Comment:
I dont want to just overwrite here, I want the original key to be recorded too.  maybe just an ehpemeral_public_key column?